staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -89,30 +106,46 @@ const (
 type FeatureGate interface {
 	// Enabled returns true if the key is enabled.
 	Enabled(key Feature) bool
+	// FeatureSpec returns the FeatureSpec of the key exists as seen by the EmulationVersion, an error otherwise
```

Comment:
```suggestion
	// FeatureSpec returns the FeatureSpec if the key exists as seen by the EmulationVersion, an error otherwise
```

Explain what "as seen by the EmulationVersion" means?